### PR TITLE
Dt/version release notes

### PR DESCRIPTION
**Pull request by toroduque**

### CODE REVIEW

File: docs/Version release notes.md [hunk 1]
```diff
@@ -7,6 +7,102 @@ pagination_prev: null
...
+- **New Destination Information:** Each `BDZoneInfo` now contains destination information accessible via `BDLocationManager.instance().zoneInfos`.
+- **Improved Architecture Support:** Migrated to `XCFramework` for both CocoaPods and Carthage, significantly improving support for multiple architectures.
+
+#### Changes
+
+- The `fences` property of `BDZoneInfo` is now non-optional in Swift.
+- Return a new structure `GeoTriggerEvent` in the `didEnterZone` and `didExitZone` callbacks that replaces `BDZoneEntryEvent` and `BDZoneExitEvent`.
+- Replaced the `onZoneInfoUpdate(zoneInfos)` callback with `didUpdateZoneInfo()`. The `zoneInfos` is no longer returned as part of this callback, access it via `BDLocationManager.instance().zoneInfos` instead.
+- Custom Event Meta Data is cleared when the SDK resets.
+
```
Comment (andrew-stirling) [inline docs/Version release notes.md:72]: Should have:
- Simplified control of "Blue Bar" background location usage indicator, using the `backgroundLocationAccessForWhileUsing` field.

Comment (toroduque) [reply docs/Version release notes.md:73]: Updated! 👍

File: docs/Version release notes.md [hunk 2]
```diff
@@ -7,6 +7,103 @@ pagination_prev: null
 Version Release Notes
 =====================
 
+Release date Feb 9, 2024
```
Comment (nehabluedot) [inline docs/Version release notes.md:10]: release date needs a update

**Review by andrew-stirling — APPROVED** [review]

File: docs/Version release notes.md (hunk 2 above)
Comment (toroduque) [reply docs/Version release notes.md]: Updated! 👍

**Pull request merged**